inc/Engine/Media/AboveTheFold/AJAX/Controller.php

```diff
@@ -209,6 +193,12 @@ function ( $item ) {
 			return null;
 		}
 
+		// Returned objects must always have a src.
+		// It is used when applying the front-end optimization.
+		if ( ! isset( $object->src ) ) {
+			$object->src = '';
+		}
```

Comment:
This is not entirely true, `bg-img` and `bg-img-set` don't use `src` on the main object.

While that doesn't break anything, I think it's good to have it noted somewhere.

Reply:
You are absolutely right @Tabrisrp. Thanks for for pointing it out. I adapted the comment. WDYT?

Reply:
yes looks good to me

Reply:
Doesn't look good to the linter :'( On it